src/sections/LocationDetails.astro

```diff
@@ -44,6 +49,9 @@ const location = {
 		{ title: 'Universal Studios', src: universal },
 		{ title: 'Knotts Berry Farm', src: farm },
 		{ title: 'Disneyland', src: disney },
+		{ title: 'Ontario Mills Outlet', src: mall },
+		{ title: 'K1 Speed — Indoor Go Karts', src: kart },
+		{ title: 'Victoria Gardents', src: vg }
 	],
```

Comment:
I believe we're _only_ featuring mall, go karts and victoria gardens. we can remove the other 3

Reply:
Done and pushed.